sakura_core/window/CEditWnd.cpp

```diff
@@ -262,7 +260,7 @@ CEditWnd::~CEditWnd()
 	delete m_pcDropTarget;	// 2008.06.20 ryoji
 	m_pcDropTarget = NULL;
 
-	// ウィンドウ毎に作成したアクセラレータテーブルを破棄する(Wine用)
+	// ウィンドウ毎に作成したアクセラレータテーブルを破棄する
```

Comment:
指摘じゃないので対処不要ですが。

```suggestion
	// アクセラレータテーブルを破棄する
```

アクセラレータテーブルの作成単位は「ウインドウごと」じゃないです。

アクセラレーターテーブルは通常、プロセス内に1つだけ存在するGUIスレッドのメッセージループで、キーボードショートカットをコマンドメッセージに変換するために使うものです。なので、メインウインドウが何枚開いていようが1つあればよいです。（状況依存とかでアクセラレーターテーブルを切り替えたい場合を除く。）

過去の開発でアクセラレーターテーブルのハンドルをプロセス間共有できてしまっていた経緯があってややこしい感じになってますが、本来的には「Wineだから共有できない」んではなく、プロセス内限定で利用するリソースです。